src/js/components/ServicesBreadcrumb.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import ServicesPathUtil from '../utils/ServicesPathUtil';
+
+const breadcrumbIcon = (
+  <i className="icon icon-sprite icon-sprite-small icon-back forward" />
```

Comment:
Although those make sense as they are configurable SVGs... What I have here is a simple assignment to avoid repeating the markup a bunch of times.